src/Raven.Server/Dashboard/DatabasesInfoRetriever.cs

```diff
@@ -142,10 +161,109 @@ public static IEnumerable<AbstractDashboardNotification> FetchDatabasesInfo(Serv
+                            var taskTag = database.WhoseTaskIsIt(topology, ravenEtlTask, processState);
+                            if (serverTag == taskTag)
+                            {
+                                ravenEtlCountAllDbs++;
+                            }
+                        }
+                        var ravenEtlCount = database.EtlLoader.RavenDestinations.Count;
+                        
+                        // SQL ETL
+                        foreach (var sqlEtlTask in database.EtlLoader.SqlDestinations)
+                        {
+                            var processState = EtlLoader.GetProcessState(sqlEtlTask.Transforms, database, sqlEtlTask.Name);
+                            var taskTag = database.WhoseTaskIsIt(topology, sqlEtlTask, processState);
+                            if (serverTag == taskTag)
+                            {
+                                sqlEtlCountAllDbs++;
+                            }
+                        }
+                        var sqlEtlCount = database.EtlLoader.SqlDestinations.Count;
                         
+                        // OLAP ETL
+                        foreach (var olapEtlTask in database.EtlLoader.OlapDestinations)
+                        {
+                            var processState = EtlLoader.GetProcessState(olapEtlTask.Transforms, database, olapEtlTask.Name);
+                            var taskTag = database.WhoseTaskIsIt(topology, olapEtlTask, processState);
+                            if (serverTag == taskTag)
+                            {
+                                olapEtlCountAllDbs++;
+                            }
+                        }
+                        var olapEtlCount = database.EtlLoader.OlapDestinations.Count;
+                        
+                        // Backup
+                        foreach (var backupTask in database.PeriodicBackupRunner.PeriodicBackups)
+                        {
+                            if (backupTask.Configuration.Name.StartsWith("Server Wide") == false)
+                            {
+                                var backupState = database.PeriodicBackupRunner.GetBackupStatus(backupTask.Configuration.TaskId);
+                                var taskTag = database.WhoseTaskIsIt(topology, backupTask.Configuration, backupState);
+                                if (serverTag == taskTag)
+                                {
+                                    periodicBackupCountAllDbs++;
+                                }
+                            }
+                        }
+                        var periodicBackupCount = database.PeriodicBackupRunner.PeriodicBackups.Count;
+                        
+                        // Subscription
+                        foreach (var keyValue in ClusterStateMachine.ReadValuesStartingWith(context, SubscriptionState.SubscriptionPrefix(database.Name)))
```

Comment:
I'm not sure about caching mechanism below. Do we still iterate through all those items even when we server values from cache? 

Should we compute: 
```
var ongoingTasksCount = extRepCount + replicationHubCount + replicationSinkCount + 
                                                ravenEtlCount + sqlEtlCount + olapEtlCount + periodicBackupCount + subscriptionCount;
```
first and then do other computations when cached version can't be used? @arekpalinski